include/classes/DAO/DAO.class.php

```diff
@@ -63,18 +65,18 @@ function DAO()
 	function execute($sql)
 	{
 		$sql = trim($sql);
-		$result = mysql_query($sql, $this->db) or die($sql . "<br />". mysql_error());
+		$result = mysqli_query( $this->db,$sql) or die($sql . "<br />". mysqli_error());
```

Comment:
`mysqli_query( $this->db,$sql)` -> `mysqli_query($this->db, $sql)`. Just to use the existing coding convention: remove the extra space in between brackets and variables; add a space in between `,` and the following variable. Thanks. 

This comment also applies to the entire pull request.